src/craftdef.h

```diff
@@ -373,10 +366,9 @@ class CraftDefinitionFuel: public CraftDefinition
 public:
 	CraftDefinitionFuel() = delete;
 	CraftDefinitionFuel(const std::string &recipe_,
```

Comment:
```suggestion
	CraftDefinitionFuel(
		const std::string &recipe_,
```

Could the first arg be moved to the next line for consistency with the rest of the changes in this file?